src/Repository/CommissionableAssetRepository.php

```diff
@@ -29,13 +29,22 @@ public function __construct(ManagerRegistry $registry)
     /**
      * @return CommissionableAsset[]
      */
-    public function search(string $query): array
+    public function search(Organization $organization, string $query): array
     {
         $words = explode(' ', $query);
-        $qb = $this->createQueryBuilder('ca')->innerJoin('ca.organization', 'o');
+        $qb = $this->createQueryBuilder('ca');
+
+        if ($organization->isParent()) {
```

Comment:
I think you can remove this test, as it's also doing an sql request in order to check if the orgranization has children or not